### PR TITLE
Fixes unmarshal errors from kubeadm init with kustomize

### PR DESCRIPTION
**Pull request by juliocri**
This fixes: #221

Removing error logs from `kubeam init` caused from mismatched json fields.

### CODE REVIEW

**Comment by juliocri** [comment]
Seems that with this flag `--experimental-kustomize` the kustomization dir is taken, but patches over controller manager are not reflected in the final config.

**Comment by jascott1** [comment]
@juliocri did you get it the kustomize to work w the corrected path? If so, what about something like:

```
KUBEADM_CONFIG="base"
if [[ -n $HIGH_POD_COUNT ]]; then
  KUBEADM_CONFIG="high-pod-count"
fi
sudo -E kubeadm init --experimental-kustomize=./kubeadm/"$KUBEADM_CONFIG"
```

**Comment by juliocri** [comment]
> @juliocri did you get it the kustomize to work w the corrected path? If so, what about something like:
> 
> ```
> KUBEADM_CONFIG="base"
> if [[ -n $HIGH_POD_COUNT ]]; then
>   KUBEADM_CONFIG="high-pod-count"
> fi
> sudo -E kubeadm init --experimental-kustomize=./kubeadm/"$KUBEADM_CONFIG"
> ```

@jascott1 no, it does not work, `--experimental-kustomize` by now only patches static pods manifests [here the doc,](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/kubeadm/20190722-Advanced-configurations-with-kubeadm-(Kustomize).md#kustomize-integration-with-kubeadm) it does not apply patches on other yamls such the kubeadm.yaml for example...

**Comment by jascott1** [comment]
Maybe `sed` out the names after templating as you suggested or just go back to `sed` altogether. :|

File: clr-k8s-examples/create_stack.sh [hunk 1]
```diff
@@ -63,6 +63,7 @@ function cluster_init() {
 		echo "/var/lib/etcd exists! skipping init."
 		return
 	fi
+	echo "$(kubectl kustomize ./kubeadm/metadata/)" > ./kubeadm/base/kubeadm.yaml
```
Comment (juliocri) [inline clr-k8s-examples/create_stack.sh]: @jascott1 a last try patching, how do this looks like? or we should move back to `sed`? as you suggest in your last comment, and let this kubeadm-kustomize gets more mature.

**Comment by juliocri** [comment]
@jascott1 this final commit removes kustomize support from kubeadm config and comes back to the `sed` replacement method. can you take a look in a chance? :)

**Pull request merged**